lib/classes/Variables.js

```diff
@@ -962,33 +962,53 @@ class Variables {
     );
   }
 
-  warnIfNotFound(variableString, valueToPopulate) {
-    if (
+  handleUnresolved(variableString, valueToPopulate) {
+    const valueEmpty =
```

Comment:
Let's rename it to `isValueEmpty` (`valueEmpty` name gives impression it's inteded to keep an empty value)